src/main/java/com/servicewizard/model/ServiceModel.java

```diff
@@ -0,0 +1,37 @@
+package com.servicewizard.model;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ServiceModel {
+
+	public String getApiName() {
+		return apiName;
+	}
+
+	public void setApiName(String apiName) {
+		this.apiName = apiName;
+	}
+
+	public String getIntroText() {
+		return introText;
+	}
+
+	public void setIntroText(String introText) {
+		this.introText = introText;
+	}
+
+	public List<Service> getServices() {
+		return services;
+	}
+
```

Comment:
I think a constructor belongs here - it's not the first thing you read in a class, but it comes before the gritty implementation details.